internal/fs/node_xattr.go

```diff
@@ -53,9 +53,10 @@ func handleXattrErr(err error) error {
 
 	case *xattr.Error:
 		// On Linux, xattr calls on files in an SMB/CIFS mount can return
-		// ENOATTR instead of ENOTSUP.
+		// ENOATTR instead of ENOTSUP.  Ignore "Operation not supported"
+		// as well.
```

Comment:
ENOTSUP is "Operation not supported"; EOPNOTSUPP is "Operation not supported on socket". So I think this comment should call out NetBSD for being weird:

```suggestion
		// ENOATTR instead of ENOTSUP.  NetBSD can return EOPNOTSUPP.
```

Reply:
From the manpage xD . Although, I agree that NetBSD shows some rather unusual behavior here.

> (ENOTSUP and EOPNOTSUPP have the same value on Linux, but according to POSIX.1 these error values should be distinct.)

Reply:
The xattr package for go:
https://github.com/pkg/xattr/blob/master/xattr_test.go
uses both ENOTSUP and EOPNOTSUPP as well.   FreeBSD (I think) will be the same as NetBSD in terms of returning EOPNOTSUPP.  Not sure about Darwin.

Reply:
EOPNOTSUPP seems indeed to be a BSD thing: https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/acl_set.3.html . Although, restic has supported xattrs on darwin, freebsd, linux and solaris for quite a while, and there haven't been any reports regarding that error code. But feel free to replace `NetBSD` with `BSD` in the above suggestion.